bundles/org.openhab.binding.boschshc/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,281 @@
+
+		<channels>
+			<channel id="latest-motion" typeId="latest-motion"/>
+		</channels>
+
+		<config-description>
+			<parameter name="id" type="text" required="true">
+				<label>Device ID</label>
+				<description>Device ID of the contact</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<thing-type id="shutter-control">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="shc"/>
+		</supported-bridge-type-refs>
+
+		<label>Shutter Control</label>
+		<description>Bosch Shutter Control</description>
+
+		<channels>
+			<channel id="level" typeId="level"/>
+		</channels>
+
+		<config-description>
+			<parameter name="id" type="text" required="true">
+				<label>Device ID</label>
+				<description>Device ID of the shutter control</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<thing-type id="thermostat">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="shc"/>
+		</supported-bridge-type-refs>
+
+		<label>Thermostat</label>
+		<description>Bosch Thermostat</description>
+
+		<channels>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="valve-tappet-position" typeId="valve-tappet-position"/>
+		</channels>
+
+		<config-description>
+			<parameter name="id" type="text" required="true">
+				<label>Device ID</label>
+				<description>Device ID of the thermostat</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<thing-type id="climate-control">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="shc"/>
+		</supported-bridge-type-refs>
+
+		<label>Climate Control</label>
+		<description>Bosch Climate Control. This is a virtual device which is automatically created for all rooms that have
+			thermostats in it.</description>
+
+		<channels>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="setpoint-temperature" typeId="setpoint-temperature"/>
+		</channels>
+
+		<config-description>
+			<parameter name="id" type="text" required="true">
+				<label>Device ID</label>
+				<description>Device ID of the climate control</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<!-- https://www.eclipse.org/smarthome/documentation/concepts/items.html -->
+	<channel-type id="power-switch">
+		<item-type>Switch</item-type>
+		<label>State</label>
+		<description>State of the Bosch Power Switch</description>
+	</channel-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Current measured temperature.</description>
+		<state min="0" max="40" step="0.5" pattern="%.1f °C" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="temperature-rating">
+		<item-type>String</item-type>
+		<label>Temperature Rating</label>
+		<description>Rating of the currently measured temperature.</description>
```

Comment:
Can you add the possible values this Channel can have, if applicable? See the `state` and `option` tags.

Reply:
Added the possible enum values of the temperature rating, the combined rating and the humidity rating.